addons/website_sale/views/templates.xml

```diff
@@ -1634,7 +1634,7 @@
                         <t t-call="website_sale.cart_line_description_following_lines">
                             <t t-set="div_class" t-valuef="d-none d-md-block"/>
                         </t>
-                        <div>
+                        <div class="o_wsale_remove_from_cart">
```

Comment:
Same comment**s** above: it's not only for the "remove from cart" button, and `class` -> `name`.
```suggestion
                        <div name="o_wsale_cart_line_button_container">
```